Makefile

```diff
@@ -37,6 +39,10 @@ ifdef ENABLE_LOCAL_USER_DIR
 CFLAGS += -DENABLE_LOCAL_USER_DIR
 endif
 
+ifdef ENABLE_PARTICLE_FX
```

Comment:
Perhaps instead of this define it's better to disable effects when the particles/ subdirectory isn't present?

Reply:
Sure.